src/System.Collections.Specialized/tests/NameValueCollection/NameValueCollection.CtorTests.cs

```diff
@@ -19,6 +20,33 @@ public void Ctor_Empty()
 
             Assert.False(((ICollection)nameValueCollection).IsSynchronized);
         }
+#if netstandard17
+        [Fact]
+        public void Ctor_Provider_Comparer()
+        {
+#pragma warning disable CS0618 // Type or member is obsolete
+            NameValueCollection nameValueCollection = new NameValueCollection(CaseInsensitiveHashCodeProvider.Default, CaseInsensitiveComparer.Default);
+#pragma warning restore CS0618 // Type or member is obsolete
+            Assert.Equal(0, nameValueCollection.Count);
+            Assert.Equal(0, nameValueCollection.Keys.Count);
+            Assert.Equal(0, nameValueCollection.AllKeys.Length);
+
+            Assert.False(((ICollection)nameValueCollection).IsSynchronized);
+        }
+
+        [Fact]
+        public void Ctor_Int_Provider_Comparer()
+        {
+#pragma warning disable CS0618 // Type or member is obsolete
+            NameValueCollection nameValueCollection = new NameValueCollection(5, CaseInsensitiveHashCodeProvider.Default, CaseInsensitiveComparer.Default);
+#pragma warning restore CS0618 // Type or member is obsolete
+            Assert.Equal(0, nameValueCollection.Count);
+            Assert.Equal(0, nameValueCollection.Keys.Count);
+            Assert.Equal(0, nameValueCollection.AllKeys.Length);
+
+            Assert.False(((ICollection)nameValueCollection).IsSynchronized);
+        }
+#endif //netstandard17
```

Comment:
Yeah, Clone methods are already there, I just added interfaces so that we match what is in netstandard.